README.md

```diff
@@ -451,6 +451,20 @@ Routing domain confederation AS. Valid values are String, keyword 'default'.
 ##### `confederation_peers`
 AS confederation parameters. Valid values are String, keyword 'default'.
 
+##### `disable_policy_batching` 
+Enable/Disable the batching evaluation of prefix advertisements to all peers. Valid values are String, keyword 'default'.
+
+##### `disable_policy_batching_prefix`
+Enable/Disable the batching evaluation of prefix advertisements to all peers with prefix list.
```

Comment:
I think the nested array unnecessarily complicates this property. This should really be two properties:

`disable_policy_batching_ipv4` and `disable_policy_batching_ipv6`, which simply take the prefix-list name string as inputs.